src/SWP/Bundle/RevisionBundle/Model/RevisionLog.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Publisher Revision Bundle.
+ *
+ * Copyright 2017 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\RevisionBundle\Model;
+
+use SWP\Component\Revision\Model\RevisionLog as BaseRevisionLog;
+use SWP\Component\Storage\Model\PersistableInterface;
+
+class RevisionLog extends BaseRevisionLog implements PersistableInterface
+{
+    /**
+     * @var int
+     */
+    protected $id;
```

Comment:
I just don't want to require storage component in so small component like revision just for that one interface.